curl_multi_info_read_basic.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+Test function curl_multi_info_read() by calling it with its required argument
+--CREDITS--
+Ryan Jentzsch ryan.jentzsch@gmail.com 
+--FILE--
+<?php
+
+//CREATE RESOURCES
```

Comment:
you should add a comment, when someone reading the code needs information about things not already obvious from the code.

writing something like
```
// this line will init curl
$ = curl_init();
```

does not provide more information.